src/test/java/com/epam/tc/hw9/PositiveApiTestsRestTrelloService.java

```diff
@@ -0,0 +1,90 @@
+package com.epam.tc.hw9;
+
+import com.epam.api.dto.BoardDto;
+import com.epam.api.dto.CardDto;
+import com.epam.api.dto.ListDto;
+import io.restassured.response.Response;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.hamcrest.Matchers.*;
+
+public class PositiveApiTestsRestTrelloService extends BaseTest {
+    @DataProvider
+    public Object[][] boardData() {
+        return new Object[][]{
+                {"FirstBoard"},
+                {"SecondBoard"},
+                {"ThirdBoard"},
+        };
+    }
+
+    @Test(description = "A test that verifies the correctness of creating a board",
+            dataProvider = "boardData")
+    public void createBoardTest(String nameBoard) {
+        boardDto.setName(nameBoard);
+        restTrelloService.createNewBoard(boardDto);
+        boards = restTrelloService.getAllBoards();
+        checkCountBoards(boards.length, defCountBoard);
+    }
+
+    @Test(description = "A test that verifies the correctness of deleting the board")
+    public void deleteBoardTest() {
+        boardDto.setName("Board for Delete");
+        restTrelloService.createNewBoard(boardDto)
+                .body("name", is("Board for Delete"));
```

Comment:
same comment as previous for not to use json parser

Reply:
+

Reply:
not all test cases are updated, but anyway - ok :(